app/presenters/content_item_presenter.rb

```diff
@@ -58,6 +58,10 @@ def taxonomy_sidebar
     @nav_helper.taxonomy_sidebar
   end
 
+  def related_navigation_sidebar
```

Comment:
I think it's worth adding a test for this as it's a public method. It could be as simple as asserting that the relevant `@nav_helper` method is called as expected.